lms/envs/common.py

```diff
@@ -2174,6 +2174,13 @@
     'DEFAULT_PAGINATION_CLASS': 'openedx.core.lib.api.paginators.DefaultPagination',
     'PAGE_SIZE': 10,
     'URL_FORMAT_OVERRIDE': None,
+    'DEFAULT_THROTTLE_CLASSES': (
```

Comment:
@saleem-latif we need to confirm that the addition of this throttling configuration will not adversely affect pre-existing APIs/endpoints -- I want to make sure we're not placing constraints on established workflows when we merge this code.